.github/workflows/ci.yml

```diff
@@ -13,13 +13,19 @@ jobs:
   ci:
     runs-on: ubuntu-latest
     steps:
+      - run: echo "node_version=`cat .nvmrc`" >> $GITHUB_ENV
       - uses: actions/checkout@v3
```

Comment:
```suggestion
      - uses: actions/checkout@v3
      - run: echo "node_version=$(cat .nvmrc)" >> $GITHUB_ENV
```

Reply:
Thanks for spotting! Fixed with https://github.com/inrupt/client-identifier-helper/pull/140/commits/834c88ea736d7776ef8c82c392d08ef7116af243